<launch>

  <arg name="camera_num" default="0" />
  <node pkg="cv_camera" type="cv_camera_node" name="camera" >
    <param name="device_id" value="$(arg camera_num)" />
    <remap from="~image_raw" to="src_image" />
  </node>

  <node pkg="opencv3mixing" type="edge_detect_node" name="edge_detector">
    <remap from="~dest_image" to="image" />
  </node>

  <node pkg="image_view2" type="image_view2" name="view" />

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" />

</launch>
